[2021-04-21 20:19:37,662] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: docker_sample.print_date 2021-04-19T21:00:00+00:00 [queued]>
[2021-04-21 20:19:37,680] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: docker_sample.print_date 2021-04-19T21:00:00+00:00 [queued]>
[2021-04-21 20:19:37,680] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-04-21 20:19:37,681] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-04-21 20:19:37,682] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-04-21 20:19:37,693] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): print_date> on 2021-04-19T21:00:00+00:00
[2021-04-21 20:19:37,706] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'docker_sample', 'print_date', '2021-04-19T21:00:00+00:00', '--job-id', '526', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dockerOperator.py', '--cfg-path', '/tmp/tmpa_cxaf_3', '--error-file', '/tmp/tmpsokwy1o5']
[2021-04-21 20:19:37,700] {standard_task_runner.py:52} INFO - Started process 1347 to run task
[2021-04-21 20:19:37,707] {standard_task_runner.py:77} INFO - Job 526: Subtask print_date
[2021-04-21 20:19:37,749] {logging_mixin.py:104} INFO - Running <TaskInstance: docker_sample.print_date 2021-04-19T21:00:00+00:00 [running]> on host 5af2b1cab26e
[2021-04-21 20:19:37,791] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=docker_sample
AIRFLOW_CTX_TASK_ID=print_date
AIRFLOW_CTX_EXECUTION_DATE=2021-04-19T21:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-04-19T21:00:00+00:00
[2021-04-21 20:19:37,792] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-04-21 20:19:37,794] {bash.py:158} INFO - Running command: date
[2021-04-21 20:19:37,804] {bash.py:169} INFO - Output:
[2021-04-21 20:19:37,806] {bash.py:173} INFO - Wed Apr 21 20:19:37 UTC 2021
[2021-04-21 20:19:37,807] {bash.py:177} INFO - Command exited with return code 0
[2021-04-21 20:19:37,832] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=docker_sample, task_id=print_date, execution_date=20210419T210000, start_date=20210421T201937, end_date=20210421T201937
[2021-04-21 20:19:37,857] {taskinstance.py:1220} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2021-04-21 20:19:37,883] {local_task_job.py:146} INFO - Task exited with return code 0
